cmd/main.go

```diff
@@ -115,11 +114,6 @@ func buildCmdTree() *cobra.Command {
 	// kubebuilder init
 	rootCmd.AddCommand(newInitCmd())
 
-	// kubebuilder update (v1 only)
-	if internal.ConfiguredAndV1() {
-		rootCmd.AddCommand(newUpdateCmd())
-	}
```

Comment:
Did you completely remove the `update` command at the end?

Reply:
See that it is not part of https://book.kubebuilder.io/migration/guide.html.
And it has not the purpose of upgrade V1 to V2

Reply:
Okey perfect, was just verifying that it was intended. You can mark as resolved.